heft-plugins/heft-isolated-typescript-transpile-plugin/tsdoc-metadata.json

```diff
@@ -0,0 +1,11 @@
+// This file is read by tools that parse documentation comments conforming to the TSDoc standard.
```

Comment:
This isn't supposed to be checked in according to its own messages. Seems like a misconfiguration in api-extractor.